packages/framework-integration-tests/integration/provider-unaware/load/artillery-executor.ts

```diff
@@ -0,0 +1,63 @@
+import * as fs from 'fs'
+import * as yaml from 'yaml'
+import * as path from 'path'
+import { ProviderTestHelper } from '@boostercloud/application-tester'
+import { runCommand } from '../../helper/run-command'
+import { CloudFormation } from 'aws-sdk'
+
+const cloudFormation = new CloudFormation()
+
+interface Phase {
+  duration: number
+  arrivalRate: number
+}
+
+interface OverrideOptions {
+  phases?: Array<Phase>
+  variables?: Record<string, any>
+}
+
+export class ArtilleryExecutor {
+  private readonly serverlessArtilleryStackPrefix = 'serverless-artillery'
+  constructor(
+    private scriptsFolder: string,
+    private readonly providerTestHelper: ProviderTestHelper,
+    private readonly stage = 'booster'
+  ) {}
+
+  public async ensureDeployed(): Promise<void> {
+    try {
+      await cloudFormation
+        .describeStacks({
+          StackName: `${this.serverlessArtilleryStackPrefix}-${this.stage}`,
+        })
+        .promise()
```

Comment:
One question. This code for load tests is provider unaware but for launching the Artillery tests it creates a resource in AWS to run from the cloud, regardless of the provider you are testing?

Reply:
Yes, because it is used to run the load test, not the Booster project, it is the same as running the integration tests on GitHub actions

Reply:
Yeah, the usage of Artillery has nothing to do with the provider you use for Booster. It is just a tool that happens to use AWS infrastructure to run. 

For example, you can execute the load tests using the local provider. This means that the server that will receive the load will be the local provider but the client generating all the requests will be Serverless Artillery, which will be hosted in AWS (which is powerful enough to generate the load).

So, we will use Serverless Artillery hosted in AWS to generate the load needed to execute the load test for any provider.

Does it make sense to you now, @davidverdu ?

Reply:
It makes sense!